WTWSMS/common/governments/WtWSMS_feudal_governments.txt

```diff
@@ -140,13 +141,6 @@ feudal_governments = {
 			bureaucratic_government
 		}
 		potential = {
-			NAND = {
-				controls_religion = yes
-				OR = {
-					religion_group = christian
-					religion_group = pagan
-				}
-			}
```

Comment:
Why? The idea that a Roman bureaucracy magically vanishes if run by Jews is laughably unrealistic!